index-reduce-v3.js

```diff
@@ -0,0 +1,58 @@
+const students = [
+	{
+		name: "Peyton",
+		grade: 47,
+	},
+	{
+		name: "Emily",
+		grade: 56,
+	},
+	{
+		name: "Claus",
+		grade: 35,
+	},
+	{
+		name: "Nelly",
+		grade: 72,
+	},
+	{
+		name: "Brandon",
+		grade: 80,
+	},
+	{
+		name: "Trent",
+		grade: 79,
+	},
+	{
+		name: "Chris",
+		grade: 61,
+	},
+	{
+		name: "James",
+		grade: 60,
+	},
+	{
+		name: "Alex",
+		grade: 86,
+	},
+	{
+		name: "Khloe",
+		grade: 97,
+	},
+];
+
+const passingScore = 60;
+const highestScore = (scoreA, scoreB) => Math.max(scoreA, scoreB);
```

Comment:
It is important that you understand what is being passed here. 
highestScore is correct that it calls Math.max and returns the highest of two scores.
However you are using highestScore as a callback to your reduce method.
When you are working with reduce, the first parameter to the callback function is the accumulator and the second is your current element. If you don't pass a second argument to reduce, you will not have any initial value for the callback. Thus, the first time in the loop, scoreA will have value undefined.